eng/SharedFramework.External.props

```diff
@@ -52,16 +52,12 @@
 
       If these are needed as direct dependencies, it is okay to change them to ExternalAspNetCoreAppReference and move up into sections above.
     -->
-    <_TransitiveExternalAspNetCoreAppReference Include="Microsoft.Win32.SystemEvents"                       Version="$(MicrosoftWin32SystemEventsVersion)" />
-    <_TransitiveExternalAspNetCoreAppReference Include="System.Drawing.Common"                              Version="$(SystemDrawingCommonVersion)" />
     <_TransitiveExternalAspNetCoreAppReference Include="System.Security.Cryptography.Pkcs"                  Version="$(SystemSecurityCryptographyPkcsVersion)" />
     <!--
-      Seems arbitrary that the following packages have their ref/ assemblies in our targeting pack but the above
-      three do not. This is likely about our public APIs i.e. external compilation requirements.
+      Seems arbitrary that the following packages have their ref/ assemblies in our targeting pack but the above assemblies
+      do not. This is likely about our public APIs i.e. external compilation requirements.
     -->
     <_TransitiveExternalAspNetCoreAppReference Include="System.Diagnostics.EventLog"                        Version="$(SystemDiagnosticsEventLogVersion)" />
-    <_TransitiveExternalAspNetCoreAppReference Include="System.Security.Permissions"                        Version="$(SystemSecurityPermissionsVersion)" />
-    <_TransitiveExternalAspNetCoreAppReference Include="System.Windows.Extensions"                          Version="$(SystemWindowsExtensionsVersion)" />
```

Comment:
Please perform similar removals in eng/Dependencies.props, eng/Version.Details.xml and eng/Versions.props. End result should be the repo only mentioning these assemblies / packages / versions in eng/PackageOverrides.txt and eng/PlatformManifest.txt (because those files describe our 5.0 release).

That said, leave eng/Version.Details.xml and eng/Versions.props alone if we must flow those versions to other repos in the dependency graph. @mmitche and @wtgodbe is there a strict coherence requirement here❔

Reply:
I don't think we need to worry about these strict coherency entries for the repos above us (SDK/installer) since they wouldn't depend on CoreFx OOB packages, but Matt might have different info

Reply:
Deleted it from more places.